metadatastore/core_v0.py

```diff
@@ -369,7 +369,8 @@ def get_events_generator(descriptor, event_col, descriptor_col,
     col = event_col
     oid = _UID_TO_OID_MAP[descriptor_uid]
     ev_cur = col.find({'descriptor_id': oid},
-                      sort=[('time', pymongo.ASCENDING)])
```

Comment:
We aren't adding a time index on events (for insert perf reasons) but I will change `descriptor` -> `descriptor_id`.